policyengine_canada/variables/gov/provinces/mb/tax/income/credits/spouse_and_common_law_partner_amount/mb_spouse_credit_amount.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_canada.model_api import *
+
+
+class mb_spouse_credit_amount(Variable):
+    value_type = float
+    entity = Household
```

Comment:
@PavelMakarchuk I'm not sure about this, does this need people test since the entity is household?